data/moves.js

```diff
@@ -14149,7 +14149,8 @@ exports.BattleMovedex = {
 			if (move.type !== 'Ground') return;
 			var target = this.activeTarget;
 			// only the attack that grounds the target ignores effectiveness
-			if (!this.runEvent('NegateImmunity', target, 'Ground')) return;
+			// if this handler is called from a chat command, and not a battle, don't ignore effectiveness
```

Comment:
Well @Slayer95 wants implementation to be moved out of this function and hardcoded as exceptions into the chat commands instead. Do you want me to keep the comment anyhow?